ccd-definition/CaseField/CaseField-JO-nonprod.json

```diff
@@ -0,0 +1,113 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "joJudgmentStatusDetails",
+    "Label": " ",
+    "FieldType": "JoJudgmentStatusDetails",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "joIsLiveJudgmentExists",
+    "Label": " ",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "judgmentDetails",
```

Comment:
Pleas modify name of variable which should indicate that its label. We can add lblJudgementDetails.

Reply:
It is the micro text displaying "Record Judgment" on top so I changed it to: lblRecordJudgment